contracts/core/Core.sol

```diff
@@ -695,11 +695,14 @@ contract Core is MasterCopyNonUpgradable, ConsensusModule, MosaicVersion, CoreSt
      *          - a validator has not already joined
      *
      * @param _validator A validator's address to join.
+     *
+     * @return The total count of joined validators.
      */
     function joinDuringCreation(address _validator)
         external
         onlyConsensus
         duringCreation
+        returns(uint256)
```

Comment:
Here I am picky (it's still according to our CS :P), however, we can use named return variable to make interface easier.

Reply:
I agree; I asked for this return to be named `minValidatorsReached` eg